app/assets/javascripts/sufia/workflow_actions_affix.js

```diff
@@ -0,0 +1,16 @@
+Blacklight.onLoad(function() {
+  $(document).on('scroll', function() {
+    var workflowDiv = $('#workflow_controls');
+
+    if ($('.workflow-actions').length) {
```

Comment:
May I suggest you move this check outside of the `$(document).on('scroll',...`, that way, it doesn't check this on every scroll action.